http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -299,6 +319,7 @@ fi
 if [ "${GPU_ENABLED}" == "true" ] ; then
   COMPOSE_FILES+=(-f ${DOCKER_COMPOSE_FILES_DIRECTORY}/`basename ${GPU_DOCKER_COMPOSE}`)
   if [ ! -z "$RSTUDIO_DOCKER_IMAGE" ] ; then
+    # Little bit of hack to switch the jupyter paths to the rstudio ones. Should be different env variables for Rstudio instead
```

Comment:
I guess it's replacing [gpu-docker-compose](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/init-resources/gpu-docker-compose.yaml). Yeah really feels like we should have separate Jupyter and RStudio files.